README.md

```diff
@@ -0,0 +1,88 @@
+release, or build the game yourself in Unity.
+
+### Downloading the latest release
+To download the latest release, go to the [releases](/releases/latest) page and
+choose the appropriate file for your platform.
+
+**Note:** The releases have been tested only on Windows 10 and Linux (Fedora 38).
+
+### Opening in Unity
+
+To open the project in Unity, you need to have the Unity editor installed.
+Clone the repository, using the following command:
+
+```bash
+git clone https://github.com/Wolokin/MagicPlatformer.git
+```
+
+Then open the project in Unity by selecting the `MagicPlatformer` folder.
+
+## Quick start
+
+### Start menu
+
+After running the game, you will be greeted with the start menu. From here you
+can select the number of players (2-4), the skin of each player and the input
+they will use.<br>
+**Note**: for the player to spawn, the input needs to be present,
+i.e. if you select a controller as an input, but there is no controller
+connected, the player will not spawn.
+
+When you're ready with your selections, proceed to select 'Map 1' at the bottom
+to start playing.
+
+### Movement & Controls
+
+The following table describes the control schemes for both keyboard and controller inputs:
+
+| Actions | Keyboard Scheme 1 | Keyboard Scheme 2 | Controller Scheme |
+|---------|-------------------|-------------------|-------------------|
+| Move    | Arrow keys        | WASD              | Left stick        |
+| Jump    | Z                 | Space             | A                 |
+| Dash    | Left Control      | Left Control      | Left Trigger      |
+| Grab    | Left Shift        | Left Shift        | Left Shoulder     |
+| Cast    | X                 | E                 | Right Shoulder    |
+
+### Pick-ups
+
+There are currently six pick-ups in total in the game. Four of them grant
+different bonuses, like health restoration and speed boost, and two of them are
+wands, that give you access to new spells.
+
+Pick-ups are spawned very often as long as there is less than 10 of them on the map.
+
+Everything is summarized in the following table:
+
+
+| Icon                            | Action                                                            |
+|---------------------------------|-------------------------------------------------------------------|
+| ![](./Images/health-potion.png) | Restores 25 health point                                          |
+| ![](./Images/mana-potion.png)   | Restores 25 mana point                                            |
+| ![](./Images/speed-boost.png)   | Increases the speed of the spell projectile by 10%                |
+| ![](./Images/dmg-boost.png)     | Increases spell's damage by 15%                                   |
+| ![](./Images/fire-wand.png)     | Replaces the current spell with an exploding fireball             |
+| ![](./Images/eternal-wand.png)  | Replaces the current spell with slow but high-damage damage spell |
```

Comment:
I will add info about how many mana and health points each player has as a maximum. Same with spell rate.

Reply:
Ok, when do you plan on adding this?

Reply:
Today, but rather later like 9 pm.

Reply:
Sounds good

To be completely honest, I was quite certain you just mistyped 'will' instead of 'would' by mistake :)